stammer.py

```diff
@@ -257,6 +287,10 @@ def main():
         return
     try:
         TEMP_DIR.mkdir()
+    except FileExistsError:
+        shutil.rmtree(TEMP_DIR)
+        TEMP_DIR.mkdir()
+    try:
```

Comment:
This just made debugging easier when I had to append sys.argv manually.  Then I didn't have to worry about the state of the `TEMP_DIR` directory.